ScalaScript.sublime-syntax

```diff
@@ -67,8 +78,24 @@ contexts:
       scope: constant.numeric.integer.scalascript
 
     # Booleans
-    - match: '(?i)\b(true|false|on|off)\b'
+    - match: '(?i)\b(True|False|On|Off)\b'
```

Comment:
Yep, still valid based on [the current syntax docs](https://docs.scala.com/display/D1102/Lexical+Rules#LexicalRules-BooleanLiterals)